src/components/accordion.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+    import type { FaqSection } from "$lib/types/components";
+    import { marked } from "marked";
+    import { slide } from "svelte/transition";
+
+    export let items: FaqSection[];
+    let activeFaq: number;
+</script>
+
+{#each items as item, index}
+    <div id={item.id}>
+        <div class="cursor-pointer" on:click={() => (activeFaq = activeFaq !== index ? index : undefined)}>
```

Comment:
its better always moving the logic out of the HTML part. Better just create a `handleClick`